src/App.js

```diff
@@ -1,21 +1,27 @@
-import Container from 'react-bootstrap/Container';
-import ItemForm from "./components/ItemForm";
 import ArchitectNextSteps from "./components/ArchitectNextSteps";
-import './index.css';
 import ArchitectHeader from './components/ArchitectHeader';
+import './index.css';
+// import ItemForm from "./components/ItemForm";
 
 const App = () => (
-    <Container className="mt-4 p-3">
+    <div className="container">
         <ArchitectHeader/>
-
-        <h1 className="header" align="center">
-            Favorite Movies
-        </h1>
-
+        {/* <h1 style={{ color: '#FF3377' }}>Uncomment me to see hot reloading in action!</h1> */}
         <ArchitectNextSteps/>
-        <ItemForm/>
-    </Container>
+        {/* <ItemForm/> */}
+    </div>
 );
 
 export default App;
 
+
+/*
+    You'll notice that we've commented out some lines of code.
+
+    If you're ready to learn more about Architect, follow the instructions
+    below to learn about additional features we've added to this application.
+
+    - Dependencies:
+        * Follow the steps in the README under "Adding a Dependency"
+        * Uncomment lines 4 and 11
+*/
```

Comment:
I like the changes. I know this can be overly nitpicky, but I find the flow of instructions from someone deploying to enabling dependencies disjointed.  `Adding a Dependency` doesn't contain steps until scrolling down to `Registering Dependencies`.  For example, my experience was:

I'm reading the comments,  then being told I would see instructions below.  The instructions tell me to read a section in the README.  I open the README and still have no steps, but another paragraph to read before I reach Registering Dependencies. 

I would love if we could adjust the amount of steps required to get from reading the initial comment `You'll notice that we've commented out some lines of code.` to making the changes if that's possible.  One idea is to have a section in the Readme about uncommenting lines 4 & 11  in App.js similar to how we instruct on modifying the architect.yml. Then our instructions become something something like: 

```
    You'll notice that we've commented out some lines of code. Follow along with 
    the "Registering Dependencies" in the README to learn more about enabling 
    some of the additional features we've included in this application.
```

Reply:
I agree, I think this is a great change and makes the UX much easier